js/widgets/widget.theme.js

```diff
@@ -29,17 +29,30 @@
 
 $.mobile.widget.theme = {
 	_create: function() {
+		var that = this;
 		this._super();
 		if ( !this.options.enhanced && this.options.theme ) {
```

Comment:
ok Im still not convinced this is a big enough problem we should worry about it. Your talking about the addition of a single class that does not effect layout only color so it still wont cause a reflow and will only repaid this one element. 

I would need to see some sort of evidence this actually makes a difference.